app/main.py

```diff
@@ -1,21 +1,28 @@
+import asyncio
 import time
+from typing import Awaitable, Any
 
 from iot.devices import HueLightDevice, SmartSpeakerDevice, SmartToiletDevice
 from iot.message import Message, MessageType
```

Comment:
Ensure that the 'Message' class has a constructor that accepts 'device_id', 'msg_type', and optionally 'data'. Also, verify that 'MessageType' includes 'SWITCH_ON', 'SWITCH_OFF', 'PLAY_SONG', 'FLUSH', and 'CLEAN'.